packages/@aws-cdk/aws-events-targets/package.json

```diff
@@ -121,6 +123,7 @@
   "stability": "stable",
   "awslint": {
     "exclude": [
+      "docs-public-apis:@aws-cdk/aws-events-targets.ApiGatewayProps.httpParameters",
```

Comment:
why not document it? - most of the linter docs exclusions are from when the linter rule was introduced. we typically shouldn't need to add any new docs exclusions.